### DISCUSSION TITLE
Why Mantine 7.0 will be released?

### DISCUSSION THREAD

**Discussion by Trav1sRen** [mantinedev/mantine#4170] [Q&A]
Read from other forum that U mentioned in Mantine 7.0 
we are gonna to use traditional CSS instead of using CSS-IN-JS with emotion. 
Does it mean it is possible to use Mantine 7.0 components in NextJS server component?
When will it be released? I am really looking forward to this change.

**Comment by rtivital** [comment]
> Why Mantine 7.0 will be released?

Uhh, well, that is the next major version, if we make breaking changes, then it will require bumping major version according to semver

> Does it mean it is possible to use Mantine 7.0 components in NextJS server component?

Yes, it will be possible to use Mantine with Next.js app dir feature without any additional setup. And no, you will not be able to use Mantine components as server components because all of them rely on React context.

> When will it be released? 

When it is ready. You can follow insights channel in Mantine Discord to learn about 7.0 updates and progress.
